### PR TITLE
simplify docs

### PR DESCRIPTION
**Pull request by zplizzi**

### CODE REVIEW

**Comment by zplizzi** [comment]
Lmk if y'all have suggestions for further simplifying the README!

**Comment by joshalbrecht** [comment]
Hrm. Still a ways to go. Could make the installation a single line to easily copy-paste for example, either with && or by making an install script. Similarly, would be good to get the to the magic moment via only copy-pasting commands, ie, provide all necessary commands including starting jupyter, editing, calling sync directly, etc. Could break instructions out for each editor.  Could have a troubleshooting FAQ as well. Etc

File: README.md [hunk 1]
```diff
@@ -1,153 +1,96 @@
 
-# jupyter_ascending
+# Jupyter Ascending
 
 Sync Jupyter Notebooks from any editor
 
 ![Jupyter Ascending](./media/simple_jupyter_ascending.gif)
```
Comment (joshalbrecht) [inline README.md:6]: This gif could be much better. It's really hard to see what is going on. Should be larger text, very little text, very short, and with very descriptive code (ex: add a line like "print('I'm writing this line in vim and it shows up in jupyter!', then sync, then add another: "print('and you can run cells too')", then sync and run, then wait at least 5 seconds so they can figure out what just happened)"

File: README.md [hunk 2]
```diff
@@ -1,153 +1,96 @@
...
 
 ## About
```
Comment (joshalbrecht) [inline README.md:8]: Better description / to replace the about section:

Jupyter Ascending lets you edit Jupyter notebooks from your favorite editor, then instantly sync and execute that code in the Jupyter notebook running in your browser.

It's the best of both worlds--the autocomplete, keybindings, and refactoring tools you love in your favorite editor, plus the great visualization abilities of a Jupyter notebook.

Combined with basic syncing of your code to a remote server, you can have all the power of a beefy dev-server with all the convenience of editing code locally.

File: README.md [hunk 3]
```diff
@@ -1,153 +1,96 @@
...
+3) Add some code to the `.sync.py` file, e.g.
 
+   `echo 'print("Hello World!")' >> example.sync.py`
+   
 
-## Usage in PyCharm
+4) Sync the code into the jupyter notebook:
 
-You'll want to set PyCharm up so that it runs `jupyter_ascending.requests.sync` every time you save a `.sync.py` file. This can be done with a File Watcher (preferences->file watcher).
+   `python -m jupyter_ascending.requests.sync --filename example.sync.py`
```
Comment (joshalbrecht) [inline README.md:52]: add one more example step 5: for executing a cell

File: README.md [hunk 4]
```diff
@@ -1,153 +1,96 @@
...
 
-![File watcher config](./media/filewatcher.png)
 
-You'll need to make a custom file watcher scope: `file:*.sync.py`
+Set up one of the editor integrations to do all of this from within your favorite editor!
+- [Visual Studio Code](docs/VSCODE.md)
+- [PyCharm](docs/PYCHARM.md)
+- [Vim](https://github.com/untitled-ai/jupyter_ascending.vim)
```
Comment (joshalbrecht) [inline README.md:58]: put vim first

File: README.md [hunk 5]
```diff
@@ -1,153 +1,96 @@
...
 
+By default the Jupyter server will search for a free port starting at 8888. If 8888 is unavailable and it selects eg 8889, Jupyter Ascending won't work - as it's expecting to connect to 8888. To force Jupyter to use a specific port, start your jupyter notebook with `JUPYTER_PORT=8888 JUPYTER_PORT_RETRIES=0 jupyter notebook` (or whatever port you want, setting also `JUPYTER_ASCENDING_EXECUTE_PORT` appropriately).
 
-## Usage in other editors
+### Working on a remote server
 
-To use Jupyter Ascending in a different editor, you just need to be able to set up a few hooks:
+Jupyter Ascending doesn't know or care if the editor and the jupyter server are on the same machine. The client is just sending requests to `http://[jupyter_server_url]:[jupyter_server_port]/jupyter_ascending`, with the default set to `http://localhost:8888/jupyter_ascending`. We typically use SSH to forward the remote jupyter port into `localhost:8888`, but you can set up the networking however you like, and use the environment variables to tell the client where to look for the Jupyter server.
```
Comment (joshalbrecht) [inline README.md:70]: add line first in this section: install on both the client and server

Comment (joshalbrecht) [reply README.md:70]: also provide an example SSH forwarding command, no need for people to figure it out from scratch!

**Comment by joshalbrecht** [comment]
Other than my other comments, LGTM!

**Review by leuasseurfarrelds247-arch — APPROVED** [review]

**Review by leuasseurfarrelds247-arch — APPROVED** [review]

**Comment by leuasseurfarrelds247-arch** [comment]
80e2615f20313d8d4e55058836b9d7096d3b1f2b

**Pull request merged**